### ISSUE TITLE
Question on channel before entering the first block

### ISSUE THREAD

**Issue by Sirius083** [liuzhuang13/DenseNet#57]
Hello, I want to reproduce the results on densenet-cifar10/cifar100, but got lower accuracy on tensorflow implementation. There is one question on model architecture,
In the paper Implementation Details part: "Before entering the first dense block, a convolutional with 16(or twice the growth eate for DenseNet-BC) output channels is performed on the input images." 
However the code seemingly all use twice the growth rate on first block.
https://github.com/liuzhuang13/DenseNet/blob/master/models/densenet.lua#L15
https://github.com/liuzhuang13/DenseNet/blob/master/models/densenet.lua#L70
Since I did not get the same accuracy level with pytorch implementaion (25.53% on cifar100 d_40_k_12_no_bottleneck, 24.42% in paper. following the same data augmentation as your official code), I am wondering whether this caused the difference?
(since I am a beginner on pytorch, maybe it is in other part of the code, can you point it out)
Thanks in advance

**Comment by ztao1991** [comment]
same issues

**Comment by YanFuHai06** [comment]
same issues
